### PR TITLE
[RFR][1LP] fix for pxe provisioning tests

### PR DESCRIPTION
**Pull request by izapolsk**
{{pytest: cfme/tests/infrastructure/test_pxe_provisioning.py}}

it also required to set System for pxe_prov_template.

### CODE REVIEW

**Comment by izapolsk** [comment]
something strange happened with tests against 5.6. but previous execution was successful.
5.7. test results look ok

File: cfme/configure/configuration/__init__.py [hunk 1]
```diff
@@ -602,8 +602,8 @@ class SMTPSettings(Updateable):
             ('port', Input("smtp_port")),
             ('domain', Input("smtp_domain")),
             ('start_tls', Input("smtp_enable_starttls_auto")),
-            ('ssl_verify', Select("select#smtp_openssl_verify_mode")),
-            ('auth', Select("select#smtp_authentication")),
+            ('ssl_verify', AngularSelect("smtp_openssl_verify_mode")),
```
Comment (RonnyPfannschmidt) [inline cfme/configure/configuration/__init__.py]: please clarify if this was an oversight, or if we need a version pick

Comment (izapolsk) [inline cfme/configure/configuration/__init__.py]: as you can see tests pass in both 5.6 and 5.7. so, I suppose oversight

Comment (RonnyPfannschmidt) [inline cfme/configure/configuration/__init__.py]: taking a look at the test results, im going to assume this one was perhaps working in 5.5 and early 5.6 and just never fixed

Comment (psav) [inline cfme/configure/configuration/__init__.py]: That looks like exactly what it was

Comment (izapolsk) [inline cfme/configure/configuration/__init__.py]: @psav, I experienced some weird error with Select and it disappeared when I switched to AngularSelect. Though those should work almost the same in this case.

**Pull request merged**